### PR TITLE
Warn in README about core functionality that currently gets broken by this extension

### PR DESCRIPTION
**Pull request by jedwards1211**
Two major issues with VSCodeVim causing core VSCode functionality not to work (or work very slowly) have wasted a lot of my time debugging.  You need to do right by users and warn them loudly about these problems up front.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,12 @@ Like real vim, VSCodeVim will take over your control keys. This behaviour can be
 
 The settings documented here are a subset of the supported settings; the full list is described in the `Contributions` tab of VSCodeVim's [extension details page](https://code.visualstudio.com/docs/editor/extension-gallery#_extension-details), which can be found in the [extensions view](https://code.visualstudio.com/docs/editor/extension-gallery) of VS Code.
 
+## WARNING
+
+Due to the amount of behavior this extension needs to modify, it may break some core VSCode functionality.  At present, VSCodeVim is known to cause the following problems:
+* Snippet placeholder transforms don't work (#3176)
+* VSCodeVim causes performance problems (#2021).  In particular, there will be a severe lag before the autocomplete menu pops up.  The only workaround at the moment is to disable all `editor.quickSuggestions` in your `settings.json`.
```
Comment (J-Fields) [inline README.md]: I haven't seen this issue. What language are you working in and what extensions do you have enabled?

**Comment by jedwards1211** [comment]
I'm having trouble finding where people first reported it and recommended disabling quick suggestions (middle of some long issue thread) but hopefully I'll find it.  In any case, TypeScript, and various other extensions but disabling VSCodeVim or disabling quick suggestions were the only things that solved the problem for me

**Comment by J-Fields** [comment]
I'd try with only VSCodeVim installed - some other extensions can hog execution time which isn't noticeable until every key stroke is blocked by that extension (which is the case when VSCodeVim is enabled).

Frankly, I'm not sure these issues are serious or widespread enough to call out explicitly in the README, and there are existing issues filed, so I'm closing this PR.

**Comment by jedwards1211** [comment]
Just made video proof of the issue: https://github.com/VSCodeVim/Vim/issues/4598

**Pull request closed without merging**